packages/core/src/components/Loader/LoaderConstants.ts

```diff
@@ -1,15 +1,21 @@
+/**
+ * @deprecated
+ */
 export enum LoaderColors {
-  PRIMARY = "var(--primary-color)",
-  SECONDARY = "var(--secondary-text-color)",
-  ON_PRIMARY = "var(--text-color-on-inverted)",
-  DARK = "var(--primary-text-color)"
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+  ON_PRIMARY = "onPrimary",
+  DARK = "dark"
 }
 
+/**
+ * @deprecated
+ */
 export enum LoaderSizes {
-  XS = 16,
-  SMALL = 24,
-  MEDIUM = 40,
-  LARGE = 64
+  XS = "xs",
+  SMALL = "small",
+  MEDIUM = "medium",
+  LARGE = "large"
 }
 
 export type LoaderSize = typeof LoaderSizes | number;
```

Comment:
Let's move it to the types file

Reply:
I added it in the types file, but forgot to remove it from here. Removed now